SamplesDashboard/SamplesDashboard/ClientApp/tsconfig.json

```diff
@@ -1,24 +0,0 @@
-{
```

Comment:
I think this file is needed for the typescript linting. Could you check again?

Reply:
Oh, true.
I've returned the file.